core/idl/wallet/common/wallet.djinni

```diff
@@ -333,8 +333,17 @@ TimePeriod = enum {
 # Class representing an account.
 Account = interface +c {
     # Key of the synchronization duration time in the synchronize event payload.
-    # The value is stored in a int 64 time expressed in miliseconds.
+    # The value is stored in a int64 time expressed in miliseconds.
     const EV_SYNC_DURATION_MS: string = "EV_SYNC_DURATION_MS";
+    # Indicates the last block height fetch during synchronization
+    # The value is stored as a int32
+    const EV_SYNC_LAST_BLOCK_HEIGHT: string = "EV_SYNC_LAST_BLOCK_HEIGHT";
```

Comment:
Super, this is our consistency point, we can specify to the client last block height.
Can you clarify here (comment) if we are talking about **block height from most recent operation** related to the account or about **last height known by explorer** at the synchronization time ?

Reply:
@RomanWlm you mean this message indicate a reorg point? I am not very clear how it works.

Reply:
We're talking about the **last block height from most recent operation** related to the account here